CHANGELOG.md

```diff
@@ -2,10 +2,6 @@
 
 # Unreleased
 
-# 2.0.4
```

Comment:
This change was probably by mistake: the 2.0.4 CHANGELOG should remain here, and there should be a new entry.